pyslackers_website/slack/templates/slack/partials/member_map.html

```diff
@@ -0,0 +1,14 @@
+{% load static %}
+<script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.2.0/leaflet.js"></script>
```

Comment:
There should be extension points in the base template to put these in the right spot (on mobile, so double check, but should be `extra_script` I think).